durabletask-azuremanaged/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# For more information on pyproject.toml, see https://peps.python.org/pep-0621/
+
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "durabletask.azuremanaged"
+version = "0.1b1"
+description = "Extensions for the Durable Task Python SDK for integrating with the Durable Task Scheduler in Azure"
+keywords = [
+    "durable",
+    "task",
+    "workflow",
+    "azure"
+]
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Programming Language :: Python :: 3",
+  "License :: OSI Approved :: MIT License",
+]
+requires-python = ">=3.9"
+license = {file = "LICENSE"}
+readme = "README.md"
+dependencies = [
+    "durabletask",
+    "azure-identity"

```

Comment:
Define a minimum required version -- likely you are not using something that was introduced recently so you can set a fairly low version, but nevertheless it's a good idea you have a minimum version. Otherwise customers / users may use existing incompatible versions or really old versions that do not have tools we might need for troubleshooting etc